opennms-container/minion/minion-config-schema.yml

```diff
@@ -123,74 +123,74 @@ categories:
   # Karaf
   - name: "karaf"
     label: "Karaf"
-    description: "TODO"
+    description: "Karaf configuration"
     categories:
       # Karaf :: Management
       - name: "management"
         label: "Management"
-        description: "TODO"
+        description: "Configuration for Karaf JMX management access"
         categories:
           # Karaf :: Management :: RMI
           - name: "rmi"
             label: "RMI"
-            description: "TODO"
+            description: "Configure remote method invocation (RMI)"
```

Comment:
Not a big deal, but this one is more of a command. Should it be "Configuration for remote method invocation (RMI)" like line 131?